webpack/assets/javascripts/react_app/components/AutoComplete/AutoCompleteActions.js

```diff
@@ -99,16 +126,18 @@ const requestSuccess = ({
   });
 };
 
-const requestFailure = ({ error, id, dispatch }) =>
+const requestFailure = ({ error, id, dispatch, isVisible = true }) => {
   dispatch({
     type: AUTO_COMPLETE_FAILURE,
     payload: {
       results: [],
-      error: error.message || error,
+      error: error.message,
```

Comment:
Can you put the full `error` object in the state? We might want to read `error.stack` as well when debugging errors.

Reply:
can use selector to get the error message from the state

Reply:
for some reason, passing the error object to the store finish with an empty object, that's why I used the `error.message`, I guess the problem that the Error object cannot be serialized.